tools/SetupFlow/DevHome.SetupFlow/Views/RepoConfigView.xaml.cs

```diff
@@ -26,6 +27,21 @@ public RepoConfigView()
     {
         relatedActivityId = Guid.NewGuid();
         this.InitializeComponent();
+
+        var selectorService = Application.Current.GetService<IThemeSelectorService>();
+        selectorService.ThemeChanged += OnThemeChanged;
+    }
+
+    public void OnThemeChanged(object sender, ElementTheme theme)

```

Comment:
Can this be "Default"? Will that confuse it as to which color is correct?

Reply:
Changed to use ActualThemeChanged.